test/test.py

```diff
@@ -29,6 +31,27 @@ def setUp(self):
         # Suppress `sys:1: ResourceWarning: unclosed <ssl.SSLSocket fd=6, family=AddressFamily.AF_INET, ...`
         warnings.simplefilter("ignore", ResourceWarning)
 
+    def test_example_bundles(self):
+        for directory, age_range, has_specimens in [
+            ('CD4+ cytotoxic T lymphocytes', AgeRange(min=567648000, max=1892160000), True),
+            ('Healthy and type 2 diabetes pancreas', AgeRange(min=1356048000, max=1356048000), True),
+            ('HPSI_human_cerebral_organoids', AgeRange(min=1419120000, max=1545264000), True),
+            ('Mouse Melanoma', AgeRange(min=3628800, max=7257600), True),
+            ('Single cell transcriptome analysis of human pancreas', AgeRange(min=662256000, max=662256000), True),
+            ('Tissue stability', AgeRange(min=1734480000, max=1892160000), False),
+            ('HPSI_human_cerebral_organoids', AgeRange(min=1419120000, max=1545264000), True),
+            ('1M Immune Cells', AgeRange(min=1639872000, max=1639872000), True)
+        ]:
+            with self.subTest(dir=directory):
+                manifest, metadata_files = download_example_bundle(repo='HumanCellAtlas/metadata-schema',
+                                                                   branch='hs-rename-beta-example-file-names-2',
```

Comment:
Change to `develop` once https://github.com/HumanCellAtlas/metadata-schema/pull/564 lands.

Reply:
@hannes-ucsc HumanCellAtlas/metadata-schema#564 has been merged! Thanks for the catch!